Sources/CoreData/Combine/CombineObjectListRepository.swift

```diff
@@ -0,0 +1,66 @@
+//
+// AppwiseCore
+// Copyright © 2022 Appwise
+//
+
+import Combine
+import CoreData
+
+public protocol CombineObjectListRepository {
```

Comment:
What does this type add? Can't we just add the `refresh()` function as an extension to the existing protocol? And have a default implementation, that wraps the call to `refresh(then:)`